taskt/UI/Forms/frmScriptBuilder.cs

```diff
@@ -496,463 +510,329 @@ private void lstScriptActions_KeyDown(object sender, KeyEventArgs e)
                 lstScriptActions_DoubleClick(null, null);
             }
             else if ((e.Control) && (e.KeyCode == Keys.X))
```

Comment:
You could also replace long chains of `else if` statements like this with a single `else if` that contains a switch, kind of like:
```
else if (e.Control) {
    switch (e.KeyCode) {
        case Keys.X:
        case Keys.C:
        ... etc
    }
}
```
Just something to be aware of, I think it improves readability. But it doesn't have to be done right now.

Reply:
I went ahead and changed this to a switch.